apixu-weatherbot1.0.tcl

```diff
@@ -243,8 +243,9 @@ namespace eval weather {
         #
         putlog "weather::_get_json was called"
 
-        if {[regexp {[^\w., ]} $location]} {
-            error "No matching location found"
+        if {[regexp {[^\w., ]} $location] && ![regexp {^[a-zA-Z]+:[a-zA-Z]+$} $location]} {
```

Comment:
Add a comment above this explaining why two regexps and what they are intending to catch.

Reply:
I can definitely do both of these changes. thanks for the review!